Completing relative prime topology (S52) (#1521)
pi-base/data: spaces/S000052/properties/P000204.md

---
space: S000052
property: P000204
value: false
---

For all $a,b,c,d \in X$, note that due to the Chinese remainder theorem, $U_a(b)$ and $U_c(d)$ are either disjoint or meet at infinitely many points.

> But if $X$ had a cut point $p$, we would find $a',b',c',d'\in X$ with $U_{a'}(b')\cap U_{c'}(d') = \{p\}$. Contradiction.

Reviewer: It seems that 𝑝 is cut point does not directly imply that there are two open sets intersecting exactly {𝑝}.

Author: Yeah I just noticed this as well... will try to fix it

---
space: S000052
property: P000204
value: false
---

> Note that for any $a,b,c,d \in X$, the closures of $U_a(b)$ and $U_c(d)$ intersect in infinitely many points.

Reviewer 2: ```suggestion
The closures of any two base elements $U_a(b)$ and $U_c(d)$ intersect in infinitely many points.
```
preferable, because saying "for any a,b,c,d in X" creates difficulties: without further restrictions, the open $U_a(b)$ does not make sense.
